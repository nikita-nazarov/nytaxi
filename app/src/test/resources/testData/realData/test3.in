1586100902000000,1586851209000000