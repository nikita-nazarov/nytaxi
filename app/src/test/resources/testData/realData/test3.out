0,2.45535186280308
1,2.61554928741093
2,2.755158221303
3,2.59715579710145
4,2.40056701030928
5,2.73814222222222
6,2.55062931034483